### PR TITLE
chore(rpc): move to a single discoverable object

### PR DESCRIPTION
**Pull request by pavelfeldman**

### CODE REVIEW

File: test/environments.js [hunk 1]
```diff
@@ -182,10 +174,25 @@ class BrowserTypeEnvironment {
...
+}
+
+class BrowserTypeEnvironment {
+  constructor(playwrightEnvironment, browserName) {
+    this._playwrightEnvironment = playwrightEnvironment;
+    this._browserName = browserName;
+  }
+
+  async beforeAll(state) {
+    state.browserType = this._playwrightEnvironment.overriddenPlaywright[this._browserName];
```
Comment (dgozman) [inline test/environments.js:195]: Why not `state.playwright[this._browserName]`? You don't need `this._playwrightEnvironment`.

File: src/rpc/channels.ts [hunk 2]
```diff
@@ -23,6 +23,16 @@ export interface Channel extends EventEmitter {
 }
 
 
+export interface PlaywrightChannel extends Channel {
+}
+export type PlaywrightInitializer = {
+  chromium: BrowserTypeChannel,
+  firefox: BrowserTypeChannel,
+  webkit: BrowserTypeChannel,
+  deviceDescriptors: types.Devices,
```
Comment (dgozman) [inline src/rpc/channels.ts:32]: Are we not going to use this in javascript client, because it is not available synchronously? If so, why do we need it?

Comment (pavelfeldman) [reply src/rpc/channels.ts:32]: It is a reference implementation.

File: test/environments.js (hunk 1 above)
Comment (pavelfeldman) [reply test/environments.js:195]: Because it is unclear what is in state.playwright!

**Pull request merged**